Gemfile

```diff
@@ -42,6 +42,7 @@ gem 'simple_form'
 gem 'sinatra', require: false
 gem 'slim-rails'
 gem 'split', require: 'split/dashboard'
+gem 'stringex'
```

Comment:
You mean https://github.com/norman/unidecoder ? I looked at it but it looks un-maintained.